docker/secondary-analysis-python/tools/file_utils/setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup
+
+setup(name='file_utils',
+      version='1.0.0.dev1',
+      description='Utilities for retrieving files from the HCA-DCP storage service',
+      url='http://github.com/HumanCellAtlas/skylab',
+      author='Saman Ehsan',
+      author_email='sehsan@broadinstitute.org',
+      license='MIT',
```

Comment:
I'm not sure if the license needs to be consistent, but skylab git repo is under `BSD 3-clause "New" or "Revised" License`, not sure if we need to modify `MIT` to `BSD 3-clause "New" or "Revised" License` here.

Reply:
Thanks for pointing that out @rexwangcc!